example/aks/main.tf

```diff
@@ -0,0 +1,114 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "~> 2.57.0"
```

Comment:
Heads up that this notation means the only acceptable versions will be 2.57.x (i.e. >= 2.57.0 && < 2.58.0). You need `"~> 2.57"` if you want >= 2.57.0 && < 3.0.0.